File: examples-sdk/go/303-rag-hybrid-exact-semantic/main.go
// 303-rag-hybrid-exact-semantic — Go SDK equivalent
// Compile: vil compile --from go --input 303-rag-hybrid-exact-semantic/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("rag-hybrid-exact-semantic", 3112)

	rag_hybrid := vil.NewService("rag-hybrid")
	rag_hybrid.Endpoint("POST", "/hybrid", "hybrid_handler")
	s.Service(rag_hybrid)

	s.Compile()
}

File: examples-sdk/zig/303-rag-hybrid-exact-semantic/main.zig
// 303-rag-hybrid-exact-semantic — Zig SDK equivalent
// Compile: vil compile --from zig --input 303-rag-hybrid-exact-semantic/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("rag-hybrid-exact-semantic", 3112);
    var rag_hybrid = vil.Service.init("rag-hybrid");
    rag_hybrid.endpoint("POST", "/hybrid", "hybrid_handler");
    server.service(&rag_hybrid);
    server.compile();
}
